neurots/generate/orientations.py

```diff
@@ -16,8 +16,12 @@
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
 import inspect
+import warnings
```

Comment:
And then you can move these into a third PR to clean it up further.

Reply:
I would not split the input/algo changes, they go together, as I said below. it does not make sense to me to run extract without any algo to use the data, I would be 'blind' in the first PR